packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -44,6 +45,29 @@ export class LinkedAccountHandler {
     }
   }
 
+  // As generated tokens expire after 6 months (180 days) per Apple documentation,
+  // generate JWT for client secret on each request instead
+  async generateAppleToken() {
+    const { clientId, keyId, privateKey, teamId } = this.config.appleAuthConfig;
+
+    if (!privateKey || !clientId || !keyId || !teamId) {
+      throw error.thirdPartyAccountError();
+    }
+    
+    const ecPrivateKey = await jose.importPKCS8(privateKey, 'ES256');
+
+    const jwt = await new jose.SignJWT({})
+      .setProtectedHeader({ alg: 'ES256', kid: keyId })
+      .setIssuedAt()
+      .setIssuer(teamId)
+      .setAudience(appleAud)
+      .setExpirationTime('2h')
```

Comment:
Since is this generated on a per request basis we could probably shorten the expiration to 1 minute or less.